src/map/script.c

```diff
@@ -13739,7 +13960,7 @@ BUILDIN(getinventorylist)
 BUILDIN(getcartinventorylist)
 {
 	struct map_session_data *sd = script->rid2sd(st);
-	char card_var[26];
+	char card_var[30];
```

Comment:
it's not really `NAME_LENGTH` -- it should rather be `SCRIPT_VARNAME_LENGTH`, since it's a variable name